microsoft/atlas-design
# Adding event listeners to popover

Task: task-533553

Link: preview-299

Adding event listeners to the Atlas' site.

## Testing

1. Go to [popover](https://design.docs.microsoft.com/pulls/299/components/popover.html) page
2. Try to open different popovers. Just one popover can be opened at a time
3. Trigger the popover. After clicking the Escape button popover should disappear, focus is on the details' summary
4. Trigger the popover. Click somewhere on a page. An opened popover should disappear
5. Trigger the popover. Using keyboard move focus away from the popover. Popover should close
6. Testing nesting popovers. Using the dev tools directly in the browser, paste one popover inside another. Opening the inner popover won't close the parent. Clicking inside the child popover content - won't close any of the popovers. Clicking the parent's popover content, while the inner one is open, would close the child popover
